src/utils/config/mixpanelDefs.ts

```diff
@@ -466,3 +466,14 @@ export type TRANSACTION_AUTH_RESP_ERROR = t.TypeOf<
 
 export const VOC_USER_EXIT = t.literal("VOC_USER_EXIT");
 export type VOC_USER_EXIT = t.TypeOf<typeof VOC_USER_EXIT>;
+
+export const NPG_INIT = t.literal("NPG_INIT");
```

Comment:
not to be fixed in this PR, but maybe we should do a refactoring to properly decouple `checkout` from `NPG`, i.e. `checkout` doesn't directly call `NPG` so also the name of the events (or the names of some variables) should refer to `ecommerce` and not just `NPG`.